Polyhedron/demo/Polyhedron/CGAL_double_edit.h

```diff
@@ -0,0 +1,28 @@
+#ifndef CGAL_DOUBLE_EDIT_H
+#define CGAL_DOUBLE_EDIT_H
+
+
+
+#include <QObject>
+#include <QtCore/qglobal.h>
+#include <QLineEdit>
+#include "Scene_config.h"
+
+
+class DoubleValidator;
+class SCENE_EXPORT DoubleEdit : public QLineEdit {
+
+  Q_OBJECT
+
+public:
+  DoubleEdit(QWidget* parent = nullptr);
+  ~DoubleEdit();
+  double value() const;
+  void setValue(double d);
+  void setMinimum(double d);
+  void setMaximum(double d);
```

Comment:
`minimum`, `maximum`, and `value` should be declared as properties (using `Q_PROPERTY`). I think that allows you to keep the compatibility with the properties from the UI file.